### ISSUE TITLE
Examples of bad text-generation

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdLiPo#17]
- Foam flecked the lips of this Ad does is add words from the metaphor of page and portal.
- This Ad that you have grown too old to blush.
- This Ad deals with the advent of electricity, was turned upside down.
- ~Lets allow the Ad swoon with delight.~

**Comment by Real-John-Cheung** [comment]
![圖片](https://user-images.githubusercontent.com/61215358/133627478-0a89bb3c-5f7d-4d37-8e57-bc9fffb3703a.png)

found when testing the responsive implementation, the "|" should not be there?

**Comment by dhowe** [comment]
> found when testing the responsive implementation, the "|" should not be there?

you are correct -- it is quite strange as 'culinary' appears only once in the database text, without any surrounding markup:

"..lies the heart of every culinary experience."
